homeassistant/components/ecowitt/sensor.py

```diff
@@ -65,7 +65,9 @@
         state_class=SensorStateClass.MEASUREMENT,
     ),
     EcoWittSensorTypes.DEGREE: SensorEntityDescription(
-        key="DEGREE", native_unit_of_measurement=DEGREE
+        key="DEGREE",
+        native_unit_of_measurement=DEGREE,
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
Does it make sense to have min/max/average on wind direction?

Reply:
No it doesn't; this is actually not valid :)

Wind direction isn't linear, but circular.